homeassistant/components/sensor/buienradar.py

```diff
@@ -185,10 +186,30 @@ def __init__(self, sensor_type, client_name):
         self._attribution = None
         self._measured = None
         self._stationname = None
+        self._unique_id = self.uuid(coordinates)
+
+        # All continuous sensors should be forced to be updated
+        self._force_update = self.type != SYMBOL and \
+            not self.type.startswith(CONDITION)
 
         if self.type.startswith(PRECIPITATION_FORECAST):
             self._timeframe = None
 
+    def uuid(self, coordinates):
+        """Generate a UUID using coordinates, client name and sensor type"""
+
+        from hashlib import sha1
+        from uuid import UUID
+
+        # The combination of the location, name an sensor type is unique
+        unique_str = "%2.6f%2.6f%s%s" % (coordinates[CONF_LATITUDE],
+                                         coordinates[CONF_LONGITUDE],
+                                         self.type,
+                                         self.client_name)
```

Comment:
Don't use a user configurable name for the unique id.

Reply:
I'll remove it from the PR then. Is there no way to make the name configurable in the front-end for these kind of sensors?

Reply:
You can, just don't use client_name as a source for unique ID. Unique IDs are not that unique if a user can change them…